### DISCUSSION TITLE
關於升級成了1.0.1版本後

### DISCUSSION THREAD

**Discussion by Elias-Lee-SC** [rime/home#1581] [Q&A] [answered]
我的鼠須管從0.18升級成了1.0.1後，之前設定的的皮膚就不能用了。現在的候選字窗口變成好小。請問怎麼回事呀？
<img width="248" alt="WechatIMG173" src="https://github.com/rime/home/assets/98945357/485be480-c6e9-4e1d-8d78-b7e64f8f5b51">
<img width="252" alt="iShot_2024-06-03_22 57 30" src="https://github.com/rime/home/assets/98945357/610c8352-0511-4993-b9ad-7f5b0d50ed36">

**Comment by LEOYoon-Tsaw** [comment, marked as answer]
設置一下字號吧，默認字號換成NSFont.systemFontSize了，不喜歡的就得自己設置自己喜歡的字號
黑暗模式下的配色必須要明確寫出color_scheme_dark才行（已在 https://github.com/rime/squirrel/commit/7fbc9111bfbabb90f79b6c5c4e6e5a5a409ce5a8 修正）

具體配置方法請看https://github.com/rime/squirrel/wiki
